sdks/java/core/src/test/java/org/apache/beam/sdk/SdkHarnessEnvironmentTest.java

```diff
@@ -74,12 +78,24 @@ public void testJammAgentAvailable() throws Exception {
   private static class TLSDoFn extends DoFn<String, String> {
     @ProcessElement
     public void processElement(ProcessContext c) {
-      assertThat(
-          Security.getProperty("jdk.tls.disabledAlgorithms").split(",[ ]*"),
-          not(hasItemInArray("TLSv1")));
-      assertThat(
-          Security.getProperty("jdk.tls.disabledAlgorithms").split(",[ ]*"),
-          not(hasItemInArray("TLSv1.1")));
+      String[] disabledAlgorithms =
+          Security.getProperty("jdk.tls.disabledAlgorithms").trim().split("\\s*,\\s*");
+      assertThat(disabledAlgorithms, not(hasItemInArray("TLSv1")));
+      assertThat(disabledAlgorithms, not(hasItemInArray("TLSv1.1")));
```

Comment:
check that they are in the legacy algorithms list so that we prefer newer versions of TLS